****
**** patches for the libtool in all the directories.
**** These are mostly for linking against the freshly build
**** libraraies rather than the installed ones.
****

--- /a/gimp-0.99.6/ltmain.sh	Wed Feb 26 03:39:50 1997
+++ ltmain.sh	Wed Mar 12 20:20:40 1997
@@ -334,7 +335,9 @@
 	  exit 1
 	  ;;
 	esac
-	deplibs="$deplibs $arg"
+	compile_shlibpath="$compile_shlibpath$dir:"
+	printf "compile_shlibpath: $compile_shlibpath\n"
+	arg=""
 	;;
 
       -l*) deplibs="$deplibs $arg" ;;
--- /a/gimp-0.99.6/gtk+/ltmain.sh	Wed Feb 26 03:39:50 1997
+++ gtk+/ltmain.sh	Wed Mar 12 20:20:40 1997
@@ -334,7 +335,9 @@
 	  exit 1
 	  ;;
 	esac
-	deplibs="$deplibs $arg"
+	compile_shlibpath="$compile_shlibpath$dir:"
+	printf "compile_shlibpath: $compile_shlibpath\n"
+	arg=""
 	;;
 
       -l*) deplibs="$deplibs $arg" ;;
--- /a/gimp-0.99.6/gtk+/glib/ltmain.sh	Wed Feb 26 03:39:50 1997
+++ gtk+/glib/ltmain.sh	Wed Mar 12 20:20:40 1997
@@ -334,7 +335,9 @@
 	  exit 1
 	  ;;
 	esac
-	deplibs="$deplibs $arg"
+	compile_shlibpath="$compile_shlibpath$dir:"
+	printf "compile_shlibpath: $compile_shlibpath\n"
+	arg=""
 	;;
 
       -l*) deplibs="$deplibs $arg" ;;
